docs/tutorials/learn.md

```diff
@@ -515,7 +515,7 @@ polymorphic definition of lists from the standard library:
 infixr 5 ::;
 type List (A : Type) :=
 | nil : List A
-| :: : A -> List A -> List A;
+| :: : {A : Type} -> A -> List A -> List A;
```

Comment:
This is wrong. That's not the definition of `::` from the standard library, and I think Juvix doesn't accept explicitly stating the constructor parameters?

Reply:
Indeed, I wanted to see the colours here, but forgot to delete it.